docs/content/stable/yugabyte-platform/manage-deployments/xcluster-replication/_index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Configure xCluster Replication for a YugabyteDB Anywhere universe
+headerTitle: xCluster Replication
+linkTitle: xCluster Replication
+description: Enable xCluster Replication between universes
+headContent: Active universe with standby using xCluster deployment
+menu:
```

Comment:
I am not sure - but should there be stable aliases too - 
aliases:
  - /stable/yugabyte-platform/create-deployments/async-replication-platform